GameDefinition/Content/CaveKind.hs

```diff
@@ -422,7 +427,7 @@ brawl = rogue  -- many random solid tiles, to break LOS, since it's a day
   , cwallTile     = "openableWall"
   , cstairFreq    = []
   , cstairAllowed = []
-  , cdesc         = ""
+  , cdesc         = "Shadows pool under the trees and leaves crunch underfoot."
```

Comment:
Yay, that so nicely underscores the main tactical feature of this (almost-tutorial) level.